GooglePay/src/main/java/com/braintreepayments/api/GooglePayInternalClient.java

```diff
@@ -18,14 +15,16 @@ void isReadyToPay(FragmentActivity activity, Configuration configuration, IsRead
                 new Wallet.WalletOptions.Builder()
                         .setEnvironment(getGooglePayEnvironment(configuration))
                         .build());
-        paymentsClient.isReadyToPay(isReadyToPayRequest).addOnCompleteListener(new OnCompleteListener<Boolean>() {
-            @Override
-            public void onComplete(@NonNull Task<Boolean> task) {
-                try {
-                    callback.onResult(task.getResult(ApiException.class), null);
-                } catch (ApiException e) {
-                    callback.onResult(false, e);
+        paymentsClient.isReadyToPay(isReadyToPayRequest).addOnCompleteListener(task -> {
+            try {
+                Boolean isReady = task.getResult(ApiException.class);
```

Comment:
Maybe this is some android syntax nuance that I'm missing, but this currently reads like:

"isReady = is there an API Exception?"
if yes, provide readiness result (which can be a failure?) **<-- can I get a confirmation of this expectation?**
otherwise not ready

Reply:
This is saying isReady = the result of the task to fetch readiness status (which will be true or false if successfully fetched), but if there is an error executing this task, then throw an APIException.

Reply:
randomtwocents: The Google Pay API is legit weird IMO. Merchant wins when we wrap! 🌯